violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/behavior/IEditorPartBehavior.java.orig

```diff
@@ -0,0 +1,249 @@
+package com.horstmann.violet.workspace.editorpart.behavior;
+
+<<<<<<< HEAD
+import java.awt.Graphics2D;
+import java.awt.event.KeyEvent;
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseWheelEvent;
+import java.awt.geom.Point2D;
+import java.beans.PropertyChangeEvent;
+
+=======
+>>>>>>> 262f0dfec02eab92605d9f9d2fa73ca26ed586a9
```

Comment:
are you doing `git add *` rather than `git add <path>/<to>/<whatever>`? Please always do the latter to avoid such problems in the future.

There's also something called ./.gitigonore which you really should use. It's useful protection against such cases.